tests/integration/test_dynamodb.py

```diff
@@ -1610,6 +1615,45 @@ def test_data_encoding_consistency(
         ]
         snapshot.match("GetRecordsAfterUpdate", records[1]["dynamodb"]["NewImage"])
 
+    @pytest.mark.skip_snapshot_verify(
+        paths=[
+            "$..PointInTimeRecoveryDescription..EarliestRestorableDateTime",
+            "$..PointInTimeRecoveryDescription..LatestRestorableDateTime",
+        ]
+    )
+    def test_continuous_backup_update(self, dynamodb_create_table, dynamodb_client, snapshot):
+        table_name = f"table-{short_uid()}"
+        dynamodb_create_table(
+            table_name=table_name,
+            partition_key=PARTITION_KEY,
+        )
+
+        def wait_for_continuous_backend():
+            try:
+                dynamodb_client.update_continuous_backups(
+                    TableName=table_name,
+                    PointInTimeRecoverySpecification=PointInTimeRecoverySpecification(
+                        PointInTimeRecoveryEnabled=True
+                    ),
+                )
+                return True
+            except ContinuousBackupsUnavailableException:
```

Comment:
Out of curiosity - under which circumstances would we run into this error here? Is this happening during creation of the DDB table, until the table status becomes `ACTIVE`? Wondering if we should add a flag to the `dynamodb_create_table` fixture to wait until the table becomes active. Also, we have a `dynamodb_wait_for_table_active` fixture, which may come in handy here?

Reply:
By default, all the tables in DDB have continuous backup enabled. However, I found that AWS takes a few seconds to enable such backups after a table has been created and is active. This basically was needed to generate the snapshot.